presto-docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -211,3 +211,6 @@ Limitations
 * Dynamic filtering is currently implemented only for :doc:`/connector/hive` and :doc:`/connector/memory` connectors.
 * Push down of dynamic filters into local table scan on worker nodes is limited to broadcast joins.
 * Min-max dynamic filter collection is not supported for DOUBLE, REAL and unorderable data types.
+* Dynamic filtering is supported in limited scenarios when join keys are of different types.
+  There should be an implicit coercion from probe key type to build key type. For example,
+  ``DOUBLE`` values can be rounded to ``REAL`` or ``INTEGER`` type, but not to ``BIGINT``.
```

Comment:
I don't understand this. Please just add an exception of coercion from probe to build type where DF would work.

Reply:
updated